matrix_sdk_base/src/models/message.rs

```diff
@@ -4,42 +4,49 @@
 //! feature is enabled.
 
 use std::cmp::Ordering;
-use std::ops::Deref;
+use std::ops::{Deref, DerefMut};
 use std::vec::IntoIter;
 
-use crate::events::room::message::MessageEvent;
-use crate::events::EventJson;
+use crate::events::AnyMessageEventStub;
 
 use serde::{de, ser, Serialize};
 
+const MESSAGE_QUEUE_CAP: usize = 35;
+
 /// A queue that holds the 10 most recent messages received from the server.
 #[derive(Clone, Debug, Default)]
 pub struct MessageQueue {
     msgs: Vec<MessageWrapper>,
 }
 
 #[derive(Clone, Debug, Serialize)]
-pub struct MessageWrapper(MessageEvent);
+pub struct MessageWrapper(pub AnyMessageEventStub);
```

Comment:
Could you document that wrapper, explaining why the wrapper is needed in the first place?